### ISSUE TITLE
Custom resolution

### ISSUE THREAD

**Issue by handbeezy** [exch-bms2/beatoraja#49]
I'd like to request support for any custom resolution (x,y) or perhaps at least upscaled 4:3 for compatability with SD LR2 skins.

Thank you.

**Comment by Agrrox** [comment]
Agree, screen detection may not be good idea, especially when using screen with weird aspect ratio, but want to use skin that was made for certain resolution.

**Comment by excln** [comment]
This will be partially improved in #78, though custom resolutions are not possible yet.

**Comment by KirbyJoe** [comment]
I agree, the custom resolution for skins are very useful to prevent the graphics "stretching" if you use a 4:3 LR2 skin in 16:9 monitor.

**Comment by excln** [comment]
On the need of aspect ratio preservation, what is expected may be the following:

* In the full-screen mode, margins should be inserted to avoid stretching of the skin.
  - It may be better that we can choose whether the skin is extended to the screen.
* In the window mode, there should be an easy way to resize the window with its aspect ratio preserved.

**Comment by phu54321** [comment]
I second this. Or at least support borderless window w/ letterboxing. This helps when playing on 16:10 aspect ratio monitor. (which is quite standard on creator-targeted windows)
